service/src/test/java/greencity/mapping/ShoppingListItemDtoMapperTest.java

```diff
@@ -0,0 +1,120 @@
+package greencity.mapping;
+
+import greencity.dto.shoppinglistitem.ShoppingListItemDto;
+import greencity.entity.ShoppingListItem;
+import greencity.entity.localization.ShoppingListItemTranslation;
+import greencity.enums.ShoppingListItemStatus;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.mockito.InjectMocks;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.stream.Stream;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.*;
+
+@ExtendWith(MockitoExtension.class)
+class ShoppingListItemDtoMapperTest {
+
+    @InjectMocks
+    private ShoppingListItemDtoMapper shoppingListItemDtoMapper;
+    private ShoppingListItemTranslation shoppingListItemTranslation;
+    private ShoppingListItem shoppingListItem;
+
+    @BeforeEach
+    void setUp() {
+        shoppingListItem = mock(ShoppingListItem.class);
+        shoppingListItemTranslation = mock(ShoppingListItemTranslation.class);
+    }
```

Comment:
could use `@Mock` annotation


```suggestion
    @Mock
    private ShoppingListItemTranslation shoppingListItemTranslation;
    @Mock
    private ShoppingListItem shoppingListItem;
```